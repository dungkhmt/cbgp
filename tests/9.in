50 50
50 100
32 49
43 39
46 24
24 48
43 34
45 8
34 4
33 44
6 0
28 4
36 7
36 13
17 5
46 11
36 9
42 6
15 8
20 1
4 5
4 6
8 5
18 17
35 11
12 17
47 11
38 20
45 14
49 4
41 46
30 21
6 2
31 1
22 16
20 27
8 9
23 17
31 12
29 24
13 26
48 39
29 3
25 19
22 2
30 19
12 44
45 10
40 16
31 21
37 12
1 2
45 3
14 3
37 0
49 2
42 22
35 14
15 9
33 17
5 34
42 16
13 1
8 1
14 11
42 27
19 0
45 7
6 1
29 11
30 25
23 37
39 32
32 4
21 19
18 23
17 0
31 19
31 0
46 29
2 3
23 12
26 7
43 32
48 11
13 9
42 20
26 3
35 47
9 7
41 48
45 9
33 31
39 2
33 8
33 12
13 7
49 39
38 42
23 0
46 48
32 34
1
48 8 10 5 34 22 38 26 62 16 57 0 42 14 36 28 2 0 33 31 0 0 48 30 15 2 32 27 43 31 31 28 37 18 55 1 51 3 43 12 28 15 45 9 34 19 51 4 48 26 42 10 37 27 33 15 76 1 44 26 44 10 12 4 55 26 9 1 82 0 44 32 34 29 50 5 31 18 53 25 35 20 49 27 32 17 55 27 12 3 37 37 48 29 45 33 52 26 54 24 
