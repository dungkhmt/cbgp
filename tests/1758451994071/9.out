(32, 49), (43, 39), (46, 24), (24, 48), (43, 34), (45, 8), (34, 4), (33, 44), (6, 0), (28, 4), (36, 7), (36, 13), (17, 5), (46, 11), (36, 9), (42, 6), (15, 8), (20, 1), (4, 5), (4, 6), (8, 5), (18, 17), (35, 11), (12, 17), (47, 11), (38, 20), (45, 14), (49, 4), (41, 46), (30, 21), (6, 2), (31, 1), (22, 16), (20, 27), (8, 9), (23, 17), (31, 12), (29, 24), (13, 26), (48, 39), (29, 3), (25, 19), (22, 2), (30, 19), (12, 44), (45, 10), (40, 16), (31, 21), (37, 12), (1, 2), (45, 3), (14, 3), (37, 0), (49, 2), (42, 22), (35, 14), (15, 9), (33, 17), (5, 34), (42, 16), (13, 1), (8, 1), (14, 11), (42, 27), (19, 0), (45, 7), (6, 1), (29, 11), (30, 25), (23, 37), (39, 32), (32, 4), (21, 19), (18, 23), (17, 0), (31, 19), (31, 0), (46, 29), (2, 3), (23, 12), (26, 7), (43, 32), (48, 11), (13, 9), (42, 20), (26, 3), (35, 47), (9, 7), (41, 48), (45, 9), (33, 31), (39, 2), (33, 8), (33, 12), (13, 7), (49, 39), (38, 42), (23, 0), (46, 48), (32, 34), 
numberIntersection=0 
minAngle=0.08490179344972226 
sumAngle=47.88064093791825 
minEdgeLength=1.4142135623730951 
sumEdgeLength=3015.423706871261 
minDistance=0.21757131728816848 
sumDistance=287863.3905029983 

0.0000000000 0.0849017934 47.8806409379 1.4142135624 3015.4237068713 0.2175713173 287863.3905029983
0: (96, 165),
1: (16, 174),
2: (65, 104),
3: (75, 87),
4: (104, 116),
5: (113, 198),
6: (79, 138),
7: (69, 83),
8: (0, 197),
9: (54, 80),
10: (186, 96),
11: (91, 76),
12: (36, 189),
13: (61, 96),
14: (88, 73),
15: (52, 106),
16: (70, 125),
17: (112, 197),
18: (103, 185),
19: (86, 153),
20: (47, 137),
21: (88, 161),
22: (68, 124),
23: (102, 181),
24: (95, 94),
25: (84, 161),
26: (70, 86),
27: (51, 134),
28: (140, 155),
29: (91, 95),
30: (86, 160),
31: (23, 185),
32: (108, 103),
33: (3, 195),
34: (159, 126),
35: (90, 71),
36: (62, 86),
37: (94, 173),
38: (52, 135),
39: (101, 96),
40: (66, 107),
41: (97, 91),
42: (58, 131),
43: (110, 95),
44: (15, 191),
45: (57, 39),
46: (95, 90),
47: (90, 73),
48: (100, 95),
49: (100, 105),
96 165 16 174 65 104 75 87 104 116 113 198 79 138 69 83 0 197 54 80 186 96 91 76 36 189 61 96 88 73 52 106 70 125 112 197 103 185 86 153 47 137 88 161 68 124 102 181 95 94 84 161 70 86 51 134 140 155 91 95 86 160 23 185 108 103 3 195 159 126 90 71 62 86 94 173 52 135 101 96 66 107 97 91 58 131 110 95 15 191 57 39 95 90 90 73 100 95 100 105 
0: (96, 165), 1: (16, 174), 2: (65, 104), 3: (75, 87), 4: (104, 116), 5: (113, 198), 6: (79, 138), 7: (69, 83), 8: (0, 197), 9: (54, 80), 10: (186, 96), 11: (91, 76), 12: (36, 189), 13: (61, 96), 14: (88, 73), 15: (52, 106), 16: (70, 125), 17: (112, 197), 18: (103, 185), 19: (86, 153), 20: (47, 137), 21: (88, 161), 22: (68, 124), 23: (102, 181), 24: (95, 94), 25: (84, 161), 26: (70, 86), 27: (51, 134), 28: (140, 155), 29: (91, 95), 30: (86, 160), 31: (23, 185), 32: (108, 103), 33: (3, 195), 34: (159, 126), 35: (90, 71), 36: (62, 86), 37: (94, 173), 38: (52, 135), 39: (101, 96), 40: (66, 107), 41: (97, 91), 42: (58, 131), 43: (110, 95), 44: (15, 191), 45: (57, 39), 46: (95, 90), 47: (90, 73), 48: (100, 95), 49: (100, 105), 
numberIntersection=0 minAngle=0.08490179344972226 sumAngle=47.880640937899834 minEdgeLength=1.4142135623730951 sumEdgeLength=3015.4237068712496 minDistance=0.21757131728816848 sumDistance=287863.3905030034 
Time taken: 34.449 s
